pages/all-features-template.json

```diff
@@ -0,0 +1,30 @@
+    
+{
+      "name":"CODENAME",
+      "models": [ "MODELNAME" ],
+      "status":"supported/experimental",
+      "maintainers": [ "MAINTAINER" ],
+      "stars":1-5,
+      "features": [
+        { "name":"Display", "status":"good/partial/bad/unknown" },
+        { "name":"Touch", "status":"good/partial/bad/unknown" },
+        { "name":"Bluetooth", "status":"good/partial/bad/unknown" },
+        { "name":"Haptics", "status":"good/partial/bad/unknown" },
+        { "name":"Tilt-to-Wake", "status":"good/partial/bad/unknown" },
+        { "name":"Ambient Light", "status":"good/partial/bad/unknown" },
+        { "name":"Always-on-Display", "status":"good/partial/bad/unknown" },
+        { "name":"Dual-Display", "status":"good/partial/bad/unknown" },
+        { "name":"Microphone", "status":"good/partial/bad/unknown" },
+        { "name":"Speaker", "status":"good/partial/bad/unknown" },
+        { "name":"Compass", "status":"good/partial/bad/unknown" },
+        { "name":"Steps", "status":"bad" },                                     // not yet implemented anywhere
```

Comment:
Although this isn't actually interpreted by anything, better would be to use the `comment` tag to show how it should be done.